apps/system/js/task_manager.js

```diff
@@ -120,7 +120,7 @@
   TaskManager.prototype.EVENT_PREFIX = 'taskmanager';
   TaskManager.prototype.name = 'TaskManager';
 
-  TaskManager.prototype.setHierarchy = function() {
+  TaskManager.prototype.setFocus = function() {
```

Comment:
Why do you just return true with any focus change?
-  Just inherited from the original behavior.

Should we create another function for setHierarchy?
- Since HierarchyManager tests the existence of "setHierarchy" every time before using it, I think it could be omitted by default.